python/dgl/utils/data.py

```diff
@@ -116,7 +116,9 @@ def networkx2tensor(nx_graph, idtype, edge_id_attr_name=None):
 SparseAdjTuple = namedtuple("SparseAdjTuple", ["format", "arrays"])
 
 
-def graphdata2tensors(data, idtype=None, bipartite=False, **kwargs):
+def graphdata2tensors(
```

Comment:
Is there any test coverage?

Reply:
There is no specific test for this method. The added arguments make it so that the sanity check is enabled by default.

Reply:
Feel free to merge if you think this is OK. I profiled the code and checked whether the GPU synchronizations are eliminated for the GraphBolt rgcn example.